### PR TITLE
[12.0] feedback Julien on m1 employee view

### PR DESCRIPTION
**Pull request by robinkeunen**
- Formatage des numéros de téléphones
- Cacher les champs hide address_id, work_location
- Retirer la page document
- Désinstaller hr_employee_emergency_contact
- Tableau éditable pour les exonérations de contribution
- retirer Create & Edit sur champs “statut à l’entrée”

### CODE REVIEW

**Review by quentinDupont — APPROVED** [review]
Review code and tested for non coopaname custom :white_check_mark:

**Comment by robinkeunen** [comment]
Thanks @quentinDupont !

File: hr_cae/models/hr_employee.py [hunk 1]
```diff
@@ -154,18 +154,3 @@ def _constrain_mutual_insurance_date(self):
                 raise ValidationError(
                     _("The start date of mutual insurance must be before the end date")
                 )
-
```
Comment (mclaeysb) [inline hr_cae/models/hr_employee.py:157]: This constraint indeed did not belong in hr_employee, but it is not present in hr_exemption yet. Let's put it there?

File: .isort.cfg [hunk 2]
```diff
@@ -9,4 +9,4 @@ line_length=88
 known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
-known_third_party=dateutil
+known_third_party=dateutil,phonenumbers
```
Comment (mclaeysb) [inline .isort.cfg:12]: Why do external libraries have to be mentioned here too?

File: coopaname_custom/__manifest__.py [hunk 3]
```diff
@@ -16,5 +19,6 @@
     ],
     "data": ["views/hr_applicant.xml", "views/hr_employee.xml", "data/data.xml"],
     "demo": [],
+    "external_dependencies": {"python": ["phonenumbers"]},
```
Comment (mclaeysb) [inline coopaname_custom/__manifest__.py:22]: Does this mean we should include it in `requirements.txt` too? What is the rule for this?

File: coopaname_custom/models/hr_employee.py [hunk 4]
```diff
@@ -5,12 +5,26 @@
...
 
+import phonenumbers
+
 from odoo import _, api, models
 from odoo.exceptions import ValidationError
 
 _logger = logging.getLogger(__name__)
 
 
+def _format_phone_number(number):
```
Comment (mclaeysb) [inline coopaname_custom/models/hr_employee.py:16]: Super interesting!

File: hr_cae/views/hr_employee.xml [hunk 5]
```diff
@@ -124,19 +132,16 @@
...
-                <page name="documents" string="Documents">
-                    <group name="passport" string="Passport">
-                        <field name="passport_id"/>
-                    </group>
-                    <group string="Work Permit" name="cae_work_permit">
-                        <field name="visa_no"/>
-                        <field name="permit_no"/>
-                        <field name="visa_expire"/>
+                        <field name="contribution_exemption_ids">
+                            <tree editable="bottom">
```
Comment (mclaeysb) [inline hr_cae/views/hr_employee.xml:136]: Why did you specify the tree here (its the same as the normal tree for this model)? To make them editable? Can't this be done in another way?

**Review by mclaeysb — APPROVED** [review]
Clean and interesting edits!

File: .isort.cfg (hunk 2 above)
Comment (robinkeunen) [reply .isort.cfg:12]: It's precommit magic. Probably for Travis.

File: coopaname_custom/__manifest__.py (hunk 3 above)
Comment (robinkeunen) [reply coopaname_custom/__manifest__.py:22]: it probably should indeed.

File: hr_cae/models/hr_employee.py (hunk 1 above)
Comment (robinkeunen) [reply hr_cae/models/hr_employee.py:157]: 🆗

File: hr_cae/views/hr_employee.xml (hunk 5 above)
Comment (robinkeunen) [reply hr_cae/views/hr_employee.xml:136]: To make it editable indeed. Not sure there is another way.

**Pull request merged**